os/board/rtl8730e/Kconfig

```diff
@@ -36,6 +36,13 @@ config BOARD_FLASH_16M
 
 endchoice
 
+config RTL8730E_REVISION
```

Comment:
1. RTL8730E_BOARD_REVISION?
2. But this may make someone mis-understand the chipset revision....

Reply:
Changed!